karyon3-admin-rxnetty/src/main/java/com/netflix/karyon/rxnetty/RxNettyAdminServerModule.java

```diff
@@ -0,0 +1,57 @@
+package com.netflix.karyon.rxnetty;
+
+import io.reactivex.netty.protocol.http.server.HttpServer;
+
+import javax.inject.Singleton;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+import com.netflix.archaius.ConfigProxyFactory;
+import com.netflix.karyon.admin.AdminModule;
+
+public class RxNettyAdminServerModule extends AbstractModule {
+    @Override
+    protected void configure() {
+        install(new AdminModule());
+        install(new RxNettyModule());
+        
+        bind(AdminView.class).annotatedWith(RxNettyAdmin.class).to(HtmlAdminView.class);
+    }
+    
+    @Provides
+    @Singleton
+    @RxNettyAdmin
+    protected HttpServer getAdminServer(@RxNettyAdmin ServerConfig config, AdminEndpointHandler handler) {
+        return HttpServer
+            .newServer(config.getServerPort())
+            .start(handler);
+    }
+    
+    // This binds our main server to Archaius configuration using the prefix
+    // 'karyon.rxnetty.shutdown'. See helloworld.properties
+    @Provides
+    @Singleton
+    @RxNettyAdmin
+    protected ServerConfig getShutdownConfig(ConfigProxyFactory factory) {
```

Comment:
Why is this called a shutdown config? It looks like the primary setting right now is the port.